package-lock.json

```diff
@@ -277,7 +277,7 @@
         "@nodejs-loaders/parse-filename": "1.0.0"
       },
       "engines": {
-        "node": ">=23.2"
+        "node": "^20.19 || ^22.12 || ^23.2"
```

Comment:
why?

Reply:
🤗🤗🤗
Just a `npm i`

Reply:
hmm? `npm i` is install. `findPackageJSON will be backported to LTS lines, so in ~2 weeks this will work on specific major-minors. Greater than or equal to 23.2 will erroneously warn this won't work on node versions were it will actually work.

Reply:
it's from https://github.com/JakobJingleheimer/nodejs-loaders/blob/ba35f4afa78675a40f6ee209d968d3c40a1abf55/packages/tsx/package.json#L18

When I `npm I`its update the package-lock.json. so it's mean the package-lock.json on main isn't correct ?

Reply:
🤦‍♂️ I read it wrong. I thought you were replacing `^20.19 || ^22.12 || ^23.2` (correct) with `>=23.2`, but it was the other way around. Sorry!